tests/Lykke.AlgoStore.Tests/Infrastructure/SettingsMock.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Threading.Tasks;
+using Lykke.SettingsReader;
+using Moq;
+
+namespace Lykke.AlgoStore.Tests.Infrastructure
+{
+    public static class SettingsMock
+    {
+        public static IReloadingManager<string> GetSettings()
+        {
+            var reloadingMock = new Mock<IReloadingManager<string>>();
+            reloadingMock
+                .Setup(x => x.Reload())
+                .Returns(() => Task.FromResult("DefaultEndpointsProtocol=https;AccountName=algostoredev;AccountKey=d2VaBHrf8h8t622KvLeTPGwRP4Dxz9DTPeBT9H3zmjcQprQ1e+Z6Sx9RDqJc+zKwlSO900fzYF2Dg6oUBVDe1w=="));
```

Comment:
Connection string is here. It's Okay?

Reply:
No remove all connection string from core.
It's must be in settings service

Reply:
This is inside test project. I am not sure how to get (load) settings inside test project so please provide me with a link to any of existing repos that have this already implemented. If there are none :) please give some suggestions on what to do here.
Thanks